ReactiveUI/Cocoa/CommonReactiveSource.cs

```diff
@@ -278,30 +234,74 @@ void resetupAll(IReadOnlyList<TSectionInfo> newSectionInfo)
             //
             // TODO: Instead of listening to Changed events and then reseting,
             // we could listen to more specific events and avoid some reloads.
-            disp.Add(sectionChanging.Subscribe(_ => {
+            disp.Add(sectionChanging.ObserveOn(ImmediateScheduler.Instance).Subscribe(_ => {
```

Comment:
Doing better than I am, I can't even build _Events.iOS64 right now, it fails telling me I need to include System.Runtime 4.0